src/ProjNet/CoordinateSystems/Projection.cs

```diff
@@ -76,6 +76,17 @@ public ProjectionParameter GetParameter(int index)
 			return _parameters[index];
 		}
 
+		/// <summary>
+		/// Adds a parameter to the parameter list
+		/// </summary>
+		/// <param name="name">The projection parameter name</param>
+        /// <param name="value">The value</param>
```

Comment:
Use `Projection.Parameters.Add(new ProjectionParameter(name, value));` and remove this method